gn3/api/metadata_api/wiki.py

```diff
@@ -66,6 +70,18 @@ def edit_wiki(comment_id: int):
                 category_addition_query, (comment_id,
                                           insert_dict["versionId"], cat_id)
             )
+
+        # Editing RDF:
+        update_wiki_comment(
+            comment_id=comment_id,
+            payload=payload,
```

Comment:
Can we pass over `insert_dict` instead to this child process? I don't like passing in the payload since `insert_dict` would be what we want to insert after doing clean up. We can modify it to work both in mysql and in rdf

Reply:
I've had to add extra keys to the `insert_dict`.